src/components/Wizard/CreateVmWizard/redux/stateUpdate/vmSettings/providers/prefillVmStateUpdate.js

```diff
@@ -28,54 +28,54 @@ import { CUSTOM_FLAVOR } from '../../../../../constants';
  * The vmwareToKubevirtOsConfigMap is usually created by the web-ui-operator and can be missing.
  */
 // update checks done in vmWareStateUpdate
-export const prefilUpdateCreator = (prevProps, prevState, props, state) => {
-  const vm = getVmwareAttribute(state, PROVIDER_VMWARE_VM_KEY, 'vm');
+export const prefilUpdateCreator = options => {
```

Comment:
Typo: The function name should be `prefillUpdateCreator`.

Reply:
fixed